### ISSUE TITLE
Every time terraform apply is run, ops agents are removed and reinstalled.

### ISSUE THREAD

**Issue by mohamedhassan7981** [terraform-google-modules/terraform-google-cloud-operations#49]
### TL;DR

When the terraform apply command is run on a project with an ops agent policy, the ops agents and their dependencies are removed from all instances that are part of that ops agent policy and then reinstalled. 
![ops-plandbad](https://user-images.githubusercontent.com/88731315/194921998-d3620090-1acc-4180-9354-0f183fb2b26a.jpg)

### Expected behavior

To not initiate ops agent removal or reinstallation.

### Observed behavior

However, once the ops agent installation and removal process begins, some servers are unable to have the ops agent reinstalled.
![bad-ops-agent](https://user-images.githubusercontent.com/88731315/194922593-6fc7b892-c932-43fb-80ad-f264b2630a2c.jpg)

### Terraform Configuration

```hcl
module "agent_policy_windows_2012" {
  source  = "terraform-google-modules/cloud-operations/google//modules/agent-policy"
  version = "~> 0.2.3"

  project_id = ""
  policy_id  = "ops-agents-windows-policy"
  agent_rules = [
    {
      type                   = "ops-agent"
      version                = "current-major"
      package_state          = "installed"
      enable_autoupgrade     = true
    },
  ]
  group_labels = [
    {
      env = "test"
      app = "windows"
    }
  ]
  os_types = [
    {
      short_name = "windows"
      version    = "6.3.9600"
    },
  ]
     
}
```

### Terraform Version

```sh
Terraform v0.13.7
```

**Comment by chalcedonyt** [comment]
Having the same issue, makes this module unusable across large numbers of VMs

**Comment by jmvanryn** [comment]
same issue.  TF does not track well things executed outside of it.. aka this module calls a local-exec of gcloud to execute the changes.

**Comment by illmaticz** [comment]
+1

**Comment by gr8jen** [comment]
Also a thing for me.
